.github/ISSUE_TEMPLATE/match_issue.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Vulnerability Match Issue
+about: Report an issue with vulnerability matching
+title: ''
+labels: [ bug, false-positive ]
```

Comment:
This may, in fact, be a `false-negative`, but it seems much more likely to get FP reports, and having two templates seems like overkill to me. Maybe having separate templates would be better 🤷